src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -175,6 +183,10 @@
 
     <ArcadeSharedFrameworkSdkOverride Include="Microsoft.DotNet.SharedFramework.Sdk" Group="ARCADE_SHARED_FX_SDK" Version="$(arcadeOutputPackageVersion)" />
     <WindowsDesktopSdkOverride Include="Microsoft.Net.Sdk.WindowsDesktop" Group="WINDOWS_DESKTOP" Location="$(ToolsDir)EmptySdk" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <NoTargetsSdkProject Include="$(SrcDir)source-build-reference-packages/**/microsoft.build.notargets.$(PrivateSourceBuiltNoTargetsSdkVersion).csproj"/>
```

Comment:
That item isn't used anywhere. Can this merged together with `NoTargetsSdkOverride`? Same question for the Traversal.

Reply:
I think it's possible - will check.

Reply:
Hmm, I don't think so. `NoTargetsSdkOverride` identity has to be `Microsoft.Build.NoTargets` for override to work. I'm using `NoTargetsSdkOverride` to do a check if specific SDK version project exists. Then from this item, I create the `NoTargetsSdkOverride` item. Similarly for Traversal.

Reply:
Ok. Can you make this exactly the same the `ArcadeSdkOverride` item and the corresponding `SourceBuiltSdkOverride` items? L172-L174 in your PR.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/27b92a3eee2f070b326230688f4dad59299358cd